diff --git a/dist/commonjs/index.js b/dist/commonjs/index.js
index f3bd634..e83994a 100644
--- a/dist/commonjs/index.js
+++ b/dist/commonjs/index.js
@@ -260,4 +260,8 @@ function expand_(str, max, maxLength, isTop) {
     }
     return acc;
 }
+module.exports = expand;
+module.exports.expand = expand;
+module.exports.EXPANSION_MAX = exports.EXPANSION_MAX;
+module.exports.EXPANSION_MAX_LENGTH = exports.EXPANSION_MAX_LENGTH;
 //# sourceMappingURL=index.js.map
